Marked tests with sanity attr

MaxGelbakhiani opened the pull request:
As a result of the review under https://github.com/nspcc-dev/neofs-s3-gw/pull/809 added sanity marks into tests

roman-khimov reviewed (changes requested):
1. No DCO
2. It doesn't fix https://github.com/nspcc-dev/neofs-s3-gw/issues/780, comment should be adjusted

MaxGelbakhiani commented:
> * No DCO
> * It doesn't fix [Setup GH workflows to run tests neofs-s3-gw#780](https://github.com/nspcc-dev/neofs-s3-gw/issues/780), comment should be adjusted

1. Fixed DCO.
2. Fixed commit message and PR description.
